src/components/Time.jsx

```diff
@@ -102,8 +102,8 @@ function Time(props) {
     if (props?.currentMap === 'factory') {
         return (
             <div className="time-wrapper">
-                <div>15:28:00</div>
-                <div>03:28:00</div>
```

Comment:
This divs are needed for correct formatting
![802DA0AA-7963-4FFC-BBB5-1E5F8B824972](https://user-images.githubusercontent.com/10927011/185229390-48a9da95-ad2e-4c70-9d64-471208206112.jpeg)